jetty-util/src/main/java/org/eclipse/jetty/util/ssl/SslSelectionDump.java

```diff
@@ -32,7 +32,7 @@
 
 public class SslSelectionDump extends ContainerLifeCycle implements Dumpable
```

Comment:
This class and its nested `CaptionedList` also should be package private classes, no need to expose them to public.